Cleaning Code/cleaning_traffic.ipynb

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
I think we can ditch this one too. The .py file seems like the one that we really want to be in our repo.